mkdocs.yml

```diff
@@ -4,14 +4,69 @@ site_url: https://emacs-lsp.github.io/lsp-mode
 nav:
   - Home: README.md
   - LSPs:
+      - Ada: lsp-ada.md
       - Apple Sourcekit: https://emacs-lsp.github.io/lsp-sourcekit
+      - Angular: lsp-angular.md
+      - Bash: lsp-bash.md
+      - C++: lsp-ccls.md
```

Comment:
I would add the name of the server in the index for the languages that have more than one server.